### ISSUE TITLE
h2load: View outgoing traffic

### ISSUE THREAD

**Issue by muety** [nghttp2/nghttp2#673]
Am I right with the assumption that h2load's "traffic" output does only refer to the data received from the server, not those that were sent?
I'm doing some benchmarks in which the request bodies contain about 1.3 kb of data and the response bodies are empty. If I do a one single request with that 1.3 kb data attached (-d data.json), the traffic is only about a hundred bytes, which doesn't make that much sense for me, does it?
So is there a way to also view the outgoing traffic in h2load?

**Comment by tatsuhiro-t** [comment]
Yes, you are right.  h2load only considers inbound traffic, and currently there is no way to show outbound traffic.
